react/features/recording/components/LiveStream/StartLiveStreamDialog.web.js

```diff
@@ -388,7 +403,10 @@ class StartLiveStreamDialog extends Component<Props, State> {
      * @returns {Promise}
      */
     _onYouTubeBroadcastIDSelected(boundStreamID) {
-        return googleApi.requestLiveStreamsForYouTubeBroadcast(boundStreamID)
+        const googleApiClient = googleApiFactory.getClient();
```

Comment:
could the factory call be done once at some earlier point rather than every time?